src/client/components/form-elements/ErrorableSelect.jsx

```diff
@@ -86,6 +88,8 @@ class ErrorableSelect extends React.Component {
 
 ErrorableSelect.propTypes = {
   errorMessage: React.PropTypes.string,
+  name: React.PropTypes.string,
+  autocomplete: React.PropTypes.string,
```

Comment:
Sorry this is really tedious, but could we also add these props in the description at the top of the file in the comments? For all the files in this PR you've touched?